### PR TITLE
added updates page content

### PR DESCRIPTION
**Pull request by haphizi**
# Description

Please include a summary of the change and where applicable, the issue you are addressing. Please also include relevant motivation and context. List any dependencies that are required for this change. If necessary, please update documentation (found under the `collections` folder) as well.

Fixes # (issue number)

## Type of change

Please check on the checkbox for those that are relevant (put x between the brackets)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Code enhancement and update (non-breaking change)
- [ ] Security patch

## How Has This Been Tested?

Please describe or list the test cases that you ran to verify your changes, and provide instructions so we can reproduce them.

### CODE REVIEW

File: contents/updates/content-updates.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Updates
+layout: layout-post
+permalink: /updates/
+---
+
+<h2>Migrating to v2</h2>
```
Comment (darenng) [inline contents/updates/content-updates.html:7]: Can change this to Upgrading to v2

File: contents/updates/content-updates.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<h6>Last updated on July 14th, 2022</h6>
+<p>
+  Yes, we are finally migrating to a new portal soon! The v2 will be released by
```
Comment (darenng) [inline contents/updates/content-updates.html:10]: Change this to "We will be upgrading to a new portal soon with a new base framework.

File: contents/updates/content-updates.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  the end of this quarter. Tune in here for further updates!
```
Comment (darenng) [inline contents/updates/content-updates.html:11]: change this to
We will be upgrading to a new portal soon with a new base framework (details below) which will be released at the end of July. Tune in here for further updates

File: contents/updates/content-updates.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        components. This will also be leveraged across upcoming projects.
+      </p>
+    </li>
+    <li>
+      <h5>React Storybook Library</h5>
+      <p>
+        In addition to our v2 portal, we will also be releasing a React library
+        version displayed in Storybook UI explorer to cater to more users.
+        Upcoming projects such as Vue & Web Component will also be hosted in
+        Storybook.
```
Comment (darenng) [inline contents/updates/content-updates.html:32]: In addition to our v2 portal, we will also be releasing a React library
version displayed in Storybook. Upcoming support for other JS frameworks will be release in the upcoming months

File: contents/updates/content-updates.html (hunk 1 above)
Comment (haphizi) [reply contents/updates/content-updates.html:7]: ok

**Review by rav3 — APPROVED** [review]

**Pull request merged**